r4/smartconfig/modules/utils/tests/test_utils_common.bal

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com). All Rights Reserved.
+
+// This software is the property of WSO2 LLC. and its suppliers, if any.
+// Dissemination of any information or reproduction of any material contained
+// herein is strictly forbidden, unless permitted by WSO2 in accordance with
+// the WSO2 Software License available at: https://wso2.com/licenses/eula/3.2
+// For specific language governing the permissions and limitations under
+// this license, please see the license as well as any agreement you’ve
+// entered into with WSO2 governing the purchase of this software and any
+// associated services.
+
+import ballerina/test;
+
+@test:Config { groups: ["Handler"] }
+function testLogHandlerDebug() returns error? {
```

Comment:
We don't need `returns error?` here

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.templates/commit/8693536cd1ee02c420a6325c46a80bc44337ed91